src/components/sync-summary/SyncSummary.jsx

```diff
@@ -2,13 +2,38 @@ import React from "react";
 import i18n from "@dhis2/d2-i18n";
 import PropTypes from "prop-types";
 import { ConfirmationDialog } from "d2-ui-components";
+import ReactJson from "react-json-view";
+
+import { withStyles } from "@material-ui/core";
+import Table from "@material-ui/core/Table";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import TableCell from "@material-ui/core/TableCell";
+import TableBody from "@material-ui/core/TableBody";
```

Comment:
Minor/irrelevant: Vscode tends to prefer `import { Table, TableHeader, ... } from "@material-ui/core";`, which looks a bit more compact.